publish-scripts/ubuntu/control_template

```diff
@@ -0,0 +1,13 @@
+Maintainer: Ahmed ElSayed <ahmels@microsoft.com>
+Version: $DEBIANVERSION
+Section: devel
+Priority: optional
+Homepage: https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local#run-azure-functions-core-tools
+Vcs-Git: https://github.com/Azure/azure-functions-core-tools.git
+Package: $PACKAGENAME
+Architecture: amd64
+Conflicts: azure-functions-core-tools, azure-functions-core-tools-2
+Replaces: azure-functions-core-tools, azure-functions-core-tools-2
```

Comment:
In case `azure-functions-core-tools` or `azure-functions-core-tools-2` is installed, this should replace them.